Tools/scripts/run_coverage.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python
+
+"""
+Runs tests with gcov coverage support.
+
+ AP_FLAKE8_CLEAN
+"""
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+os.environ['PYTHONUNBUFFERED'] = '1'
+
+tools_dir = os.path.dirname(os.path.realpath(__file__))
+root_dir = os.path.realpath(os.path.join(tools_dir, '../..'))
+
+
+class CoverageRunner(object):
+    """Coverage Runner Class."""
+
+    def __init__(self):
+        """Init method.
```

Comment:
```suggestion
        """
```

This goes without saying.